server/src/resolvers.js

```diff
@@ -49,6 +49,7 @@ export const Resolvers = {
             username: username,
             version: 1,
           })).then((user) => {
+            user.setOrganisation(1); //default org for now
```

Comment:
pretty sure this returns a promise so we need to return that so it can be awaited by graphql. I'll rewrite this whole section in #19.